contracts/interfaces/IEverlongEvents.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.20;
+
+interface IEverlongEvents {
+    /// Admin ///
```

Comment:
I'm just trying to understand the new kind of comment that looks like this:

```
    // ╭─────────────────────────────────────────────────────────╮
    // │ Storage                                                 │
    // ╰─────────────────────────────────────────────────────────╯
```

I like those comments a lot, and it seems like a good way to distinguish sections vs sub-sections in the code. In other places, we have one of these even though the section is clear from context (we do this with `view` in a few places and `storage`). With this in mind, it would make sense to me to add a similar style comment for `Events` here.

I guess what I'm really asking for is some kind of consistent style that we'll use for these comments.

Reply:
Initial thinking was that we'd use 

```
    // ╭─────────────────────────────────────────────────────────╮
    // │ Storage                                                 │
    // ╰─────────────────────────────────────────────────────────╯
```
on all contracts for storage+constructor+public+internal....

then use 
```
/// Admin ///
```

for contracts that bring together various slices of functionality (base contracts, storage contracts (don't have one yet but probably will, unified read interfaces etc...)

Reply:
That makes a lot of sense. I like it!